APIs/SentinelHub/Byoc/Examples.qmd

```diff
@@ -1,622 +1,182 @@
-
--   Store your raster data in the cloud optimized geotiff (COG)
-    [format](https://www.cogeo.org/){target="_blank"} on your own S3
-    bucket in the supported region.
--   Configure the bucket\'s permissions so that Sentinel Hub can read
-    them.
--   Import tiles using the Dashboard or API.
-
-Your data needs to be organized into collections of tiles. Each tile
-needs to contain a set of bands and (optionally) an acquisition date and
-time. Tiles with the same bands can be grouped into collections. Think
-of the Sentinel-2 data as a collection of Sentinel-2 tiles.
-
-#### A note about COG overviews used for processing
-
-When processing data, we select the nearest overview level which has
-higher resolution than your request, or the full resolution image.
-
-### Sentinel Hub BYOC Tool
-
-The Sentinel Hub BYOC Tool is software which can be used to prepare your
-data for use in Sentinel Hub. It can be run either in Docker or as a
-Java JAR. It takes care of the entire process; it is simple to use for
-simple cases but is also highly configurable allowing for more complex
-requirements. The same steps can be done manually and are detailed
-below, should you prefer or require more control over the process. Get
-the tool
-[here](https://github.com/sentinel-hub/byoc-tool){target="_blank"}.
-
-## Converting to COG
-
-### Constraints and settings
-
-COGs can contain either a single band or multiple bands. For multi-band
-COGs we support both [planar configurations
-formats](https://www.awaresystems.be/imaging/tiff/tifftags/planarconfiguration.html){target="_blank"} -
-chunky and planar format.
-
-There are a few additional constraints in addition to having COG files.
-These are:
-
--   The COG header size must not exceed one megabyte.
--   The internal tile size must be between 256 x 256 and 2048 x 2048.
--   The projection needs to be one of: WGS84 (EPSG:4326), WebMercator
-    (EPGS:3857), any UTM zone (EPSG:32601-32660, 32701-32760), or Europe
-    LAEA (EPSG:3035).
--   The COG must not cross any of the two poles.
--   The band name should be a valid JavaScript identifier so it can be
-    safely used in evalscripts; valid identifiers are case-sensitive,
-    can contain Unicode letters, \$, \_, and digits (0-9), but may not
-    start with a digit, and should not be one of the reserved JavaScript
-    keywords.
--   There can be at most 100 bands.
--   Multi-band COGs in chunky format can have at most 10 bands.
--   The file names need to be consistent for all tiles in a collection.
-    For example, if you have B1.tiff in one tile then you also need
-    B1.tiff in all the other tiles in your collection.
--   All files of each tile needs to have consistent extension (so a tile
-    cannot contain both B1.tiff and B2.TIF).
--   The maximum allowable difference between the intersection of all
-    file bounding boxes and each individual file is one pixel of that
-    file \[1\].
```

Comment:
Instead of \[1\], lets keep it as a footnote.

Maybe something like here: https://quarto.org/docs/authoring/footnotes-and-citations.html#:~:text=Style%20for%20bibliography-,Footnotes,-Pandoc%20supports%20numbering